tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1

```diff
@@ -104,37 +108,80 @@ try{
             $buildParams['PSModuleRestore'] = $true
         }
 
+        if ($Configuration -eq 'min-size')
+        {
+            $buildParams['ForMinimalSize'] = $true
+        }
+
         Start-PSBuild -Clean -Runtime $Runtime -Configuration Release @releaseTagParam @buildParams
     }
 
-    if ($Runtime -eq 'fxdependent')
+    if ($ComponentRegistration.IsPresent)
     {
-        $pspackageParams = @{'Type'='fxdependent'}
+        Write-Verbose "Exporting project.assets files ..." -Verbose
+
+        $projectAssetsCounter = 1
+        $projectAssetsFolder = Join-Path -Path $destination -ChildPath 'projectAssets'
+        $projectAssetsZip = Join-Path -Path $destination -ChildPath 'windowsProjectAssetssymbols.zip'
+        Get-ChildItem $location\project.assets.json -Recurse | ForEach-Object {
+            $subfolder = $_.FullName.Replace($location,'')
+            $subfolder.Replace('project.assets.json','')
+            $itemDestination = Join-Path -Path $projectAssetsFolder -ChildPath $subfolder
+                    New-Item -Path $itemDestination -ItemType Directory -Force
+            $file = $_.FullName
+            Write-Verbose "Copying $file to $itemDestination" -Verbose
+            Copy-Item -Path $file -Destination "$itemDestination\" -Force
+            $projectAssetsCounter++
```

Comment:
Where does this get used?

Reply:
I'm not sure. This was existing code, see below: https://github.com/PowerShell/PowerShell/blob/877c97fdc39af0c156fdfc6d0edd341abd64be0f/tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1#L173-L186

@TravisEz13, shall we remove `$projectAssetsCounter`?